docker/parabol-ubi/docker-host-st/docker-compose.yaml

```diff
@@ -1,51 +1,144 @@
-version: '3.7'
+version: '3.9'
 
 services:
-  db:
-    image: rethinkdb:latest
+  postgres:
+    container_name: postgres
+    image: postgres:15.4
     restart: always
+    env_file: .env
+    environment:
+      - PGUSER=$POSTGRES_USER
     ports:
-      - '8080:8080'
-      - '29015:29015'
-      - '28015:28015'
+      - '5432:5432'
     volumes:
-      - ./rethink-data:/data
+      #- './postgres/postgres.conf:/usr/local/etc/postgres/postgres.conf'
+      - './data/postgres/pgdata:/var/lib/postgresql/data'
+    #command: 'postgres -c config_file=/usr/local/etc/postgres/postgres.conf'
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-d", "$POSTGRES_DB", "-U", "$POSTGRES_USER"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
     networks:
       - parabol-network
-  postgres:
-    image: postgres:15.4
-    restart: always
+  pgadmin:
+    container_name: pgadmin
+    image: dpage/pgadmin4:8.3
+    depends_on:
+      postgres:
+        condition: service_healthy
     env_file: .env
     ports:
-      - '5432:5432'
+      - "5050:80"
+    networks:
+      - parabol-network
+  rethinkdb:
+    container_name: rethinkdb
+    image: rethinkdb:2.4.2
+    restart: always
+    ports:
+      - '8080:8080'
+      - '29015:29015'
+      - '28015:28015'
     volumes:
-      - './postgres.conf:/usr/local/etc/postgres/postgres.conf'
-      - './postgres-data/pgdata:/var/lib/postgresql/data'
-    command: 'postgres -c config_file=/usr/local/etc/postgres/postgres.conf'
+      - ./data/rethink:/data
     networks:
       - parabol-network
   redis:
-    image: redis
+    container_name: redis
+    image: redis:7.0-alpine
+    healthcheck:
+      test: "[ $$(redis-cli ping) = 'PONG' ]"
+      interval: 10s
+      timeout: 5s
+      retries: 5
     restart: always
     ports:
       - '6379:6379'
     volumes:
-      - ./redis-data:/data
+      - ./data/redis:/data
+    networks:
+      - parabol-network
+  pre-deploy:
+    container_name: pre-deploy
+    image: us-central1-docker.pkg.dev/prbl-tooling/parabol-production/parabol:v7.15.2
+    command: bash -c "node dist/preDeploy.js"
+    env_file: .env
+    environment:
+      - SERVER_ID=0
+    volumes:
+      - './.env:/parabol/.env'
+    depends_on:
+      rethinkdb:
+        condition: service_started
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
     networks:
       - parabol-network
-  app:
-    image: #image:tag
+  # chronos:
```

Comment:
+1 could we use profiles instead of commenting this out? https://docs.docker.com/compose/profiles/ then the client could just start 1 or many profiles

Reply:
Nice that. I've never seen that. I'm a newbie on Docker compose 😸 I'll be checking it. Thanks!